### DISCUSSION TITLE
Add support for specifying number formats when importing master measures from Excel file

### DISCUSSION THREAD

**Discussion by mountaindude** [ptarmiganlabs/ctrl-q#316] [Ideas]
Similar to how colors can be specified in the source Excel file, it would be useful to be able to define number formats too.

It might be possible  to make the process easier by using pre-defined dropdown in some columns in the source Excel file (instead of specifying a JSON defining the number format).
The downside would be
- Difficult to get international number formats correct. 
  For example, how would Ctrl-Q know whether to use DD/MM/YY, YYYY-MM-DD or something else as date format?
- Ctrl-Q would have to do more  parsing of the definitions in the Excel file. Not a problem per se, but increased risk of bugs.
- It would not be possible to directly import a master-items definitions file that has been exported from QS using Ctrl-Q. 
  This one is tricky and needs some thought.

**Comment by gianfrdp** [comment]
Hello,
was this option implemented?
I tried to export master measures from one app, export includes number format
`{"qType":"F","qnDec":2,"qUseThou":0,"qFmt":"#,##0","qDec":".","qThou":","}`
Is it possible to include this format expression in excel to import in another/same app?
